--- orig/makefile	2017-09-05 10:46:33.375508800 +0100
+++ makefile	2017-09-05 10:53:45.139306500 +0100
@@ -7,11 +7,11 @@
 KMC_TOOLS_DIR = kmc_tools
 
 CC 	= g++
-CFLAGS	= -Wall -O3 -m64 -static -Wl,--whole-archive -lpthread -Wl,--no-whole-archive -std=c++11 
-CLINK	= -lm -static -O3 -Wl,--whole-archive -lpthread -Wl,--no-whole-archive -std=c++11 
+CFLAGS	= -Wall -O3 -m64 -Wl,--whole-archive -lpthread -Wl,--no-whole-archive -std=c++11 
+CLINK	= -lm -O3 -Wl,--whole-archive -lpthread -Wl,--no-whole-archive -std=c++11 
 
-KMC_TOOLS_CFLAGS	= -Wall -O3 -m64 -static -Wl,--whole-archive -lpthread -Wl,--no-whole-archive -std=c++14
-KMC_TOOLS_CLINK	= -lm -static -O3 -Wl,--whole-archive -lpthread -Wl,--no-whole-archive -std=c++14
+KMC_TOOLS_CFLAGS	= -Wall -O3 -m64 -Wl,--whole-archive -lpthread -Wl,--no-whole-archive -std=c++14
+KMC_TOOLS_CLINK	= -lm -O3 -Wl,--whole-archive -lpthread -Wl,--no-whole-archive -std=c++14
 
 DISABLE_ASMLIB = false
 
