--- a/makeMappabilityMask.py	1970-01-01 00:00:01.000000000 +0000
+++ b/makeMappabilityMask.py	2017-12-07 16:15:13.113870000 +0000
@@ -23,11 +23,11 @@
       self.lastStartPos = pos
       self.lastCalledPos = pos
 
-with open("/lustre/scratch113/projects/msmc/ref/human_g1k_v37.mask_35_50.fa", "r") as f:
+with open(sys.argv[1], "r") as f:
   for line in f:
     if line.startswith('>'):
       chr = line.split()[0][1:]
-      mask = MaskGenerator("/lustre/scratch113/projects/msmc/ref/masks/hs37d5_chr{}.mask.bed.gz".format(chr), chr)
+      mask = MaskGenerator(sys.argv[1] + "_chr{}.mask.bed.gz".format(chr), chr)
       pos = 0
       continue
     for c in line.strip():
